grails-app/services/org/pih/warehouse/importer/InventoryLevelImportDataService.groovy

```diff
@@ -152,13 +155,19 @@ class InventoryLevelImportDataService implements ImportDataService {
     InventoryLevel updateOrCreateInventoryLevel(Location facility, Product product, Map params) {
         // FIXME This should handle duplicate inventory level but I'm not exactly sure
         //  what to do at this point so I'm going to let the user correct duplicates on their own
-        InventoryLevel inventoryLevel = findOrCreateInventoryLevel(facility, product)
+        InventoryLevel inventoryLevel = InventoryLevel.findByInventoryAndProductAndInternalLocationIsNull(facility.inventory, product)
+        if (!inventoryLevel) {
+            inventoryLevel = new InventoryLevel()
+        }
 
+        inventoryLevel.inventory = facility.inventory
+        inventoryLevel.product = product
         inventoryLevel.status = params.status ? params.status as InventoryStatus : InventoryStatus.SUPPORTED
         inventoryLevel.abcClass = params.abcClass
         inventoryLevel.minQuantity = params.minQuantity as Integer
         inventoryLevel.reorderQuantity = params.reorderQuantity as Integer
         inventoryLevel.maxQuantity = params.maxQuantity as Integer
+        inventoryLevel.forecastPeriodDays = 30
```

Comment:
why is this needed? Looks like it defaults to 30 in the domain. Are you just wanting to be more explicit about it here?

Reply:
Yeah, as soon as I pushed the PR I realized I should have removed it. The reason I added it was because I was logging the inventory level properties while debugging the earlier issue
```
log.info "inventory level " + inventoryLevel.properties
```
and this was causing a division by zero error because one of the transient properties was resolving the denominator to 0. Line 170 didn't actually resolve the issue, but I forgot to remove it. Will do so now.